velero/backup/common-service/label-bedrock-role.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: label-bedrock-job
```

```suggestion
  name: label-bedrock-job
  namespace: <every namespace in the tenant>
```

Do we need to add namespace here, since adopters need to create roles and rbs under different namespaces. If there is a namespace in this template, they do not need to project to a different ns and apply roles every time.

---

I agree, it will probably be easier to leave the namespace blank than to edit the namespace each time